src/Uno.UI/Microsoft/UI/Xaml/Controls/IconSource/BitmapIconSource.cs

```diff
@@ -27,9 +27,9 @@ public bool ShowAsMonochrome
 		}
 
 		public static DependencyProperty ShowAsMonochromeProperty { get; } =
-			DependencyProperty.Register(nameof(ShowAsMonochrome), typeof(bool), typeof(BitmapIconSource), new PropertyMetadata(default(bool)));
+			DependencyProperty.Register(nameof(ShowAsMonochrome), typeof(bool), typeof(BitmapIconSource), new PropertyMetadata(default(bool), OnPropertyChanged));
 
-		public override IconElement CreateIconElement()
+		internal protected override IconElement CreateIconElementCore()
```

Comment:
should this be `internal protected` (ie public) or `private protected` (private)?